Port upstream changes & OR support

Silarn opened the pull request:
Additions to the IPluginList in January are breaking this on recent dev versions.

This downstreams the new functions and adds the new columns to the plugin display.

And I added Oblivion Remastered to the list of supported games.

Silarn committed:
Port upstream changes & OR support

Silarn committed:
Add support for modDataDirectory

Exit-9B commented:
Can you fix this build error?
```
D:\a\modorganizer-bsplugins\modorganizer-bsplugins\build\modorganizer_super\modorganizer-bsplugins\src\MOTools\LootGroups.cpp(99,8): error C2039: 'commit': is not a member of 'MOBase::SafeWriteFile' [D:\a\modorganizer-bsplugins\modorganizer-bsplugins\build\modorganizer_super\modorganizer-bsplugins\vsbuild\src\bsplugins.vcxproj]
      D:\a\modorganizer-bsplugins\modorganizer-bsplugins\build\modorganizer_super\uibase\src\safewritefile.h(35,18):
      see declaration of 'MOBase::SafeWriteFile'
```
It looks like `SafeWriteFile` now requires `->` to access the `commit` function.

Silarn committed:
Upstream fixes
- Utilize vcpkg build
- Use native python
- Fix bsatk include
- Update SafeWriteFile
- Use strerror_s

Silarn commented:
@Exit-9B Didn't notice your response, but yes that was another recent change. I just reconfigured it to work with the new vcpkg build environment.

Holt59 commented:
The CI will fail since the new build system is not compatible with this version of the plugin, you need to update the workflow to build manually (without the `build-with-mob-action`) or update this repository to use VCPKG.

Exit-9B commented:
The build succeeded, but the install step failed due to the manually specified `modorganizer_super` directory which doesn't seem to be part of the build path anymore.
